### DISCUSSION TITLE
Is it possible to define setup file for deno test or deno run?

### DISCUSSION THREAD

**Discussion by krutoo** [denoland/deno#27276] [Q&A] [answered]
Hi, I want to try to define some tests for my project using `jsdom` or `happy-dom`

Is it possible to add setup file for deno test? Like in Jest for example?

**Comment by marvinhagemeister** [comment, marked as answer]
No, the built in test runner that's invoked when you call `deno test` doesn't support defining a setup file. There is a feature request for that here https://github.com/denoland/deno/issues/26988
